src/main/java/edu/kis/vh/nursery/stacks/IntLinkedList.java

```diff
@@ -46,25 +49,28 @@ public boolean isFull() {
 	 * @return -1, gdy lista jest pusta
 	 * @return last.value, gdy element istnieje
 	 */
+	@Override
 	public int top() {
 		if (isEmpty())
-			return EMPTY;
+			return INITIAL;
 		return last.value;
 	}
 	/**
 	 * Metoda usuwająca ostatni element
 	 * @return -1, gdy lista jest pusta
 	 * @return ret, zwraca usuniętą wartość
 	 */
+	@Override
 	public int pop() {
 		size--;
 		if (isEmpty())
-			return EMPTY;
+			return INITIAL;
```

Comment:
You should use EMPTY here (the name is too short to make this constant understandable)